l10n_it_account_stamp/models/invoice.py

```diff
@@ -29,10 +32,11 @@ def is_tax_stamp_applicable(self):
 
     @api.onchange('tax_line_ids')
     def _onchange_tax_line_ids(self):
-        if self.is_tax_stamp_applicable():
-            self.tax_stamp = True
-        else:
-            self.tax_stamp = False
+        if self.auto_compute_stamp:
+            if self.is_tax_stamp_applicable():
```

Comment:
You could improve this part as follows: `self.tax_stamp = self.is_tax_stamp_applicable()`